resources/helm/overlays/istio_cni/templates/clusterrolebinding.yaml

```diff
@@ -1,4 +1,4 @@
-{{ if .Values.enabled }}
+{{ if .Values.cni.enabled }}
```

Comment:
Do we still need an overlay file? Can't we just use upstream's from istio repo?

Reply:
We do. We run multiple containers in the istio-cni-node Pods (one for each CNI version).